torch/_dynamo/variables/functions.py

```diff
@@ -1571,7 +1594,14 @@ def call_function(
             return variables.UserDefinedClassVariable(
                 value, mutation_type=ValueMutationNew()
             )
-        unimplemented("namedtuple with non constant args")
+        unimplemented_v2(
+            gb_type="namedtuple construction",
```

Comment:
I've been using "constant" loosely throughout the graph break messages - perhaps we can just mention constants here? We might want to define what a constant is more precisely somewhere (we do have a clear code definition for it).

Reply:
This is subtle -- when a user sees "constant", they likely think of actually constant objects like string, integer, tuple, etc. But in our code, it's VT that implements `as_python_constant`, which doesn't really seem to have any constraint. For instance `ListVariable` implements it, but is a list constant...? So I decided to leave it vague here.